### ISSUE TITLE
Add a custom field type - [6h-12h]

### ISSUE THREAD

**Issue by rootux** [Metaburn/doocrate#248]
# Description
As an project admin - we can set custom fields. Currently, they are only free text.
(See image from editing a project to see the current state) - תאריך, שם ההתארגנות 
<img width="721" alt="Screen Shot 2020-08-12 at 17 08 02" src="https://user-images.githubusercontent.com/1032526/90025143-684d1300-dcbe-11ea-920c-2f9866d59c2c.png">

We want to support adding custom field types - specifically "Multiple choice" (select) with known entries.
Also we want to be able to later on filter by that custom field.

TLDR - 
- Add custom field type
- We can set it to be “Multiple choice”
- Default is “Short Text”
- We can checkbox whether or not to filter by it

Show on front page - should be "Allow filtering" - Seting this to true should show this field under the filter area
<img width="646" alt="Screen Shot 2020-08-12 at 15 36 50" src="https://user-images.githubusercontent.com/1032526/90015986-04bce880-dcb2-11ea-9f60-890cb75897a7.png">
